src/commands/environments/update.ts

```diff
@@ -31,35 +31,59 @@ export default class EnvironmentUpdate extends Command {
   };
 
   static args = [{
-    name: 'name',
+    name: 'environment',
     description: 'Name of the environment to update',
+    required: true,
     parse: (value: string) => value.toLowerCase(),
+  }, {
+    name: 'account_name',
+    description: 'Account that the environment belongs to',
+    required: false
```

Comment:
This feels like a strange syntax. Since we already force services to be "namespaced" via their own names, perhaps this should just be a single param in the format `{account_name}/{env_name}` to match how we reference services.